ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-chain.yaml

```diff
@@ -4,4 +4,4 @@ chain:
   - ref: ipi-conf
   - ref: ipi-conf-aws
   documentation: |-
-    The IPI configure step chain generates the install-config.yaml file based on the cluster profile and optional input files.
+    The IPI configure step chain generates the install-config.yaml file based on the cluster profile and optional input files.
```

Comment:
probably drop this change, so the rehearsal engine doesn't pull in all IPI AWS jobs ;).